scripts/ec2.py

```diff
@@ -78,13 +84,17 @@ def main():
         sqs_reply = sqs.receive_message(
```

Comment:
When there's no messages, this while loop will be a very tight loop, and so might max out the CPU (maybe?!)  Should we add `WaitTimeSeconds=60` to force `sqs.receive_message` to wait up to 60 seconds if the queue is empty?  [Docs here](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.receive_message).

```suggestion
        sqs_reply = sqs.receive_message(
            WaitTimeSeconds=60,
```

Reply:
great point!